src/assets.rs

```diff
@@ -23,13 +23,6 @@ pub fn register(app: &mut bevy::prelude::App) {
         .add_asset_loader(EguiFontLoader);
 }
 
-// An error that could ocurr during asset processing
-#[derive(thiserror::Error, Debug)]
-pub enum AssetLoaderError {
```

Comment:
Just checking to make sure this is no longer necessary or useful..

Reply:
Yeah, it wasn't actually used. It was used in the code I had that I copied the old loader from, but it didn't end up being incorporated.